### PR TITLE
Add an example of Composer 3 configuration with network attachment

### PR DESCRIPTION
**Pull request by spapi17**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown [hunk 1]
```diff
@@ -302,6 +301,70 @@ resource "google_project_iam_member" "composer-worker" {
 }
 ```
 
+### Cloud Composer 3 networking configuration
+
+In Cloud Composer 3, networking configuration is simplified compared to
+previous versions. You don't need to specify network ranges, and can attach
+custom VPC networks to your envrionment.
+
+<Note>
```
Comment (melinath) [inline mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown:310]: I'm pretty sure this HTML element doesn't do anything useful - you might want to rewrite this to use supported callouts instead: https://googlecloudplatform.github.io/magic-modules/develop/handwritten-docs-style-guide/#callouts. You can preview the results in Hashicorp's docs preview tool: https://registry.terraform.io/tools/doc-preview

File: mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown [hunk 2]
```diff
@@ -302,6 +301,70 @@ resource "google_project_iam_member" "composer-worker" {
 }
 ```
 
+### Cloud Composer 3 networking configuration
+
+In Cloud Composer 3, networking configuration is simplified compared to
+previous versions. You don't need to specify network ranges, and can attach
+custom VPC networks to your envrionment.
```
Comment (melinath) [inline mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown:308]:
```suggestion
custom VPC networks to your environment.
```

File: mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown [hunk 3]
```diff
@@ -302,6 +301,70 @@ resource "google_project_iam_member" "composer-worker" {
...
+
+<Note>
+  It's not possible to detach a VPC network using Terraform. Instead, you can
+  attach a different VPC network in its place, or detach the network using
+  other tools like Google Cloud CLI.
+</Note>
+
+Switch to Private IP networking:
```
Comment (melinath) [inline mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown:316]:
```suggestion
Use Private IP networking:
```

**Review by melinath — CHANGES_REQUESTED** [review]
a couple small suggestions, otherwise looks good. Please feel free to fix formatting for all the notes in this PR if you like.

File: mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown (hunk 1 above)
Comment (spapi17) [reply mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown]: fixed the callouts

**Review by melinath — APPROVED** [review]

**Pull request merged**